src/main/java/com/javastudy/coworkings/dao/jdbc/mapper/UserMapper.java

```diff
@@ -0,0 +1,43 @@
+package com.javastudy.coworkings.dao.jdbc.mapper;
+
+import com.javastudy.coworkings.entity.User;
+import com.javastudy.coworkings.entity.UserRole;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class UserMapper {
+
+    public User rowMap(ResultSet resultSet) {
+        try {
+            //Not sure if we need to separate getting values from ResultSet and assigning
+            //them via builder. If needed, I can refactor like this: setId(resultSet.getLong("id")
```

Comment:
No matters, this is not an error in both cases, it's all about your codding style. But it'll be better when all mappers have the same style.
Another thing, we are not pushing comments into master)